src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/IRemoteTagHelperProviderService.cs

```diff
@@ -12,5 +12,7 @@ namespace Microsoft.CodeAnalysis.Remote.Razor
     internal interface IRemoteTagHelperProviderService
     {
         ValueTask<TagHelperResolutionResult> GetTagHelpersAsync(RazorPinnedSolutionInfoWrapper solutionInfo, ProjectSnapshotHandle projectHandle, string factoryTypeName, CancellationToken cancellationToken);
```

Comment:
We could technically remove this endpoint entirely; however I'm letting it stick around for now until we have more of the delta tech proven in real user scenarios